cli/azd/pkg/project/service_target_aks.go

```diff
@@ -578,3 +575,40 @@ func (t *aksTarget) setK8sContext(ctx context.Context, serviceConfig *ServiceCon
 
 	return nil
 }
+
+// resolveClusterName attempts to resolve the cluster name from the following sources:
+// 1. The 'AZD_AKS_CLUSTER' environment variable
```

Comment:
Please add a changelog entry for customers to know about this new env-vars (this PR and https://github.com/Azure/azure-dev/pull/3251/files)

Also, let's make sure it gets mentioned in the official docs.

I would even explore if we can have this information as part of some azd --help command.

Reply:
The env var is not new here, adding additional configuration to pull from azure.yaml, etc.  Will check to ensure these new configuration options are available in docs.